src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGetProjectKind.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+namespace NuGet.VisualStudio.Internal.Contracts
+{
+    public enum NuGetProjectKind
+    {
+        Classic,
+        ProjectK,
+        MSBuild,
```

Comment:
>MSBuild [](start = 8, length = 7)

this might be a good opportunity to have good names. Would love to phrase them in logical terms -- PackagesConfig, PackageReference, etc... if possible? perhaps we talk though the meanings of them all.  ProjectK is ancient...and would be better if we didn't need...but am ok with leaving in...so we don't add more issues.

Reply:
If you can give me better names for them I will update it, I just used what the initial type we were checking off of

Reply:
Will revisit this in a later PR.